ng-client/src/styles.scss

```diff
@@ -8,9 +8,6 @@ body {
 }
 
 .main-container {
-  margin-left: auto;
-  margin-right: auto;
-  max-width: 1024px;
```

Comment:
This ended up looking pretty weird with navigation. I'm comfortable saying that we're targeting mobile devices due to our adoption of material.